lib/cc/analyzer/issue_location_existence_validation.rb

```diff
@@ -0,0 +1,56 @@
+module CC
+  module Analyzer
+    class IssueLocationExistenceValidation < Validation
+      def valid?
+        if location["lines"]
+          valid_lines?
+        elsif location["positions"]
+          valid_position?(location["positions"]["begin"]) &&
+            valid_position?(location["positions"]["end"])
+        end
+      end
+
+      def message
+        "Location does not exist in the file"
+      end
```

Comment:
Eh, I don't see a big win in that - do you? It has to be a method in order to conform to the `Validator` format